### DISCUSSION TITLE
892851C8-8C0B-5274-CF23-541591998C5D

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#136] [General]
# 892851C8-8C0B-5274-CF23-541591998C5D

Planets and moons generated by StarGen for Cham stars in IC 3094 (Incomplete due to a typo in Stargen), M 31, and Ring Universe. Planets in Ring Universe have no moons due to the fact it would have generated a file more than 6 gigabytes heavy.

https://celestia.mobi/addon?item=892851C8-8C0B-5274-CF23-541591998C5D

**Comment by Lil-Gdops** [comment]
Why are the planets squished and some moons are basically the same size as the planet?
